docs/modules/chalk.md

```diff
@@ -0,0 +1,35 @@
+# chalk
+
+`chalk` and similar libraries have various alternatives which are lighter and faster.
```

Comment:
we should probably mention `cli-color` here somewhere since we're pointing to this page for it. otherwise I think it could be a bit confusing
```suggestion
`chalk`, `cli-color`, and similar libraries have various alternatives which are lighter and faster.
```